user/details/script.js

```diff
@@ -704,73 +704,96 @@ function generatePRsTabDetails() {
     type: 'div',
     classList: ['user-pr'],
   });
+
   const pagination = createElement({ type: 'div', classList: ['pagination'] });
   const prevBtn = createElement({
     type: 'button',
-    classList: ['pagination-load-more'],
+    classList: ['pagination-prev-page', 'btn'],
+  });
+  prevBtn.appendChild(createTextNode('Prev'));
+  prevBtn.addEventListener('click', () => {
+    if (currentPageIndex > 1) {
```

Comment:
`Suggestion(for next PR)`: Use early return 

```js
if currentPageIndex < 1 return
currentPageIndex--;
getUserPrs();
```

Reply:
ok